core/templates/services/image-upload-helper.service.ts

```diff
@@ -29,6 +30,11 @@ angular.module('oppia').factory('ImageUploadHelperService', [
         // Separate out the mime component.
         var mime = dataURI.split(',')[0].split(':')[1].split(';')[0];
 
+        // If the data component is of svg+xml mime type, it must be
+        // sanitized.
+        if (mime === 'image/svg+xml') {
+          byteString = $sanitize(byteString.replace(/\n|\r/g, ' '));
```

Comment:
Interesting... Can we turn allowSvg on for this usage only rather than globally?